### PR TITLE
Sl/5262/bug ordered pairs fix

### PR DESCRIPTION
**Pull request by sirgeika**

### CODE REVIEW

File: test/test-bug-ordered_pairs.lua [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+local test =
+{
+  [{ h = { } }] = '87',
+  [{ b = { } }] = '66',
+}
+ 
+for k, v in ordered_pairs(test) do 
+  print(k, v)
+end
```
Comment (vladfedin) [inline test/test-bug-ordered_pairs.lua]: No newline in the end of file.

Comment (agladysh) [inline test/test-bug-ordered_pairs.lua]: @sirgeika please make sure that you have git hooks enabled for this project.

**Comment by agladysh** [comment]
@sirgeika please add yourself to AUTHORS.

File: test/test-bug-ordered_pairs.lua [hunk 2]
```diff
@@ -0,0 +1,19 @@
+-- Test based on real bug scenario -- https://github.com/lua-nucleo/lua-nucleo/issues/31
```
Comment (agladysh) [inline test/test-bug-ordered_pairs.lua]: This file should not be here. Please follow the way the rest of the tests are done.

BTW, if you would add a new test file (not sure if you should do that) do not forget to regenerate the list of tests.

File: test/test-bug-ordered_pairs.lua [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+dofile('../lua-nucleo/import.lua')
```
Comment (agladysh) [inline test/test-bug-ordered_pairs.lua]: Please use the test suite. See all other test cases.

**Comment by agladysh** [comment]
Not accepted.

**Pull request closed without merging**